.github/workflows/tests.yml

```diff
@@ -64,17 +65,7 @@ jobs:
         with:
           ruby-version: "2.7.1"
           bundler: "2.1.4"
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: ~/vendor/bundle
-          key: ${{ runner.os }}-${{ matrix.ruby }}-gems-${{ hashFiles('apps/*/Gemfile.lock', 'Gemfile.lock') }}
-
-      - name: Setup Bundler
-        run: |
-          bundle config path ~/vendor/bundle
-          bundle install
+          bundler-cache: true
```

Comment:
e2e tests don't need all the gems in the app directories, so this cache (cache key) may get us into trouble. Let's simplify and just use the `bundler-cache` feature and cache the gems only the top level `Rakefile` needs.